etl/scripts/get_data_from_staging.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+# change working dir
+cd /home/mseith
+
+# remove old data
+rm -r staging_data
+
+# make data dir
+mkdir staging_data
+
+# switch dir
+cd staging_data
+
+# remove old data file
+rm enhanced_output.tar.gz
+
+# get new file from staging
+wget -4 http://cai-einander-helfen-staging.fbi.h-da.de/enhanced_output.tar.gz
+
+# extract data from folder
+tar xfvz enhanced_output.tar.gz
+
+# delete old data
+rm -r /home/mseith/einander-helfen/etl/data_enhancement/data
```

Comment:
This will only work if the user that executes this code is mseith
Please use a neutral directory for the ETL process.

Reply:
my bad, will fix now